### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by henrycadenaec**
This branch was created to practice coding review. I have added some coding comments, to obtain more tools to figure out how I can comment and sign possible errors or suggestions.

Please consider: The webhint error (not successful checked) will not be taken into account by the reviewers. Thank you very much.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,11 @@
     <main id="main">
       <h1 id="title">Dr. Norman Borlaug</h1>
       <figure id="img-div">
+        <!-- <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+        /> -->
```
Comment (henrycadenaec) [inline index.html:18]: Kindly, remove this comment, as it is not a good practice. It indicates that your code is not clean code.

File: index.html [hunk 2]
```diff
@@ -30,6 +35,9 @@ <h3>
             target="_blank"
             >Get more info</a
           >
+          <!-- Example code:
+            <button type="button">Example</button>
+           -->
```
Comment (henrycadenaec) [inline index.html:40]: Kindly, remove this comment, as it is not a good practice. It indicates that your code is not clean code.

File: index.html [hunk 3]
```diff
@@ -11,6 +11,11 @@
     <main id="main">
       <h1 id="title">Dr. Norman Borlaug</h1>
       <figure id="img-div">
+        <!-- <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+        /> -->
         <img
           style="display: block; max-width: 100%"
```
Comment (henrycadenaec) [inline index.html:20]: I suggest that you place all the styles in the style sheet. That will make your code cleaner and more organized.

**Review by henrycadenaec — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f7651474244546b687449756e614a596d52462f67697068792d646f776e73697a65642e6769663f6369643d323963616361373573663375713979347167756c6b357962626f6a77

**Highlights**
Correct use of Gitflow ✔️
Linter checks are passed ✔️
Good PR description ✔️
Meaningful commit messages ✔️
**Required Changes ♻️**
Check the comments under the review.

**Review by souad988 — APPROVED** [review]
Hi @henrycadenaec , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
use inline comments
Use polite communication
Add review summary

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henrycadenaec** [comment]
Thank you very much @souad988 !!!

**Pull request merged**